src/coreclr/vm/jitinterface.cpp

```diff
@@ -9997,12 +9997,15 @@ namespace
             _ASSERTE(pMD->IsNDirect() || pMD->HasUnmanagedCallersOnlyAttribute());
             if (pMD->IsNDirect())
             {
-                if (pSuppressGCTransition)
-                {
-                    *pSuppressGCTransition = pMD->ShouldSuppressGCTransition();
-                }
+#ifdef CROSSGEN_COMPILE
+                // Return CorInfoCallConvExtension::Managed to indicate that crossgen does not support handling UnmanagedCallConv
+                if (pMD->HasUnmanagedCallConvAttribute())
+                    return CorInfoCallConvExtension::Managed;
```

Comment:
Does the JIT handle the case when the "Managed" callconv is returned from here correctly? (It should decide that it cannot inline the call as an unmanaged call).

Reply:
Yeah, `impCheckForPInvokeCall` checks for `CorInfoCallConvExtension::Managed` as indication that the unmanaged calling convention couldn't be determined and bails without inlining the call.